### PR TITLE
Add initial config.

### PR DESCRIPTION
**Pull request by Vodzoo**

### CODE REVIEW

File: src/app/core/env-config.service.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import { EMPTY, Observable, tap } from 'rxjs';
+import { environment } from '../../environments/environment';
+
+export interface EnvConfig {}
+
+@Injectable({
+  providedIn: 'root',
+})
+export class EnvConfigService {
+  static readonly CONFIGURATION_FILE_NAME: string = 'envConfig.json';
```
Comment (sfc-gh-krzymkowski) [inline src/app/core/env-config.service.ts:13]: Shouldn't we have a default `envConfig.json`? Won't it throw 404 error in production build?

Comment (Vodzoo) [reply src/app/core/env-config.service.ts:14]: This file should be delivered when deploying on different environments.
I think app should not be able to start when there is no config file as it's crucial for app to work properly.

**Comment by neu5** [comment]
@bbartosiak co z tym PR? Już nieaktualne i zamykamy?

**Comment by bbartosiak** [comment]
do wyjaśnienia co chcemy tam trzymać

**Pull request closed without merging**